CMake/External_Qt.cmake

```diff
@@ -252,14 +252,28 @@ add_dependencies(Download Qt-download)
 
 fletch_external_project_force_install(PACKAGE Qt)
 
-set(QT_QMAKE_EXECUTABLE ${fletch_BUILD_INSTALL_PREFIX}/bin/qmake
-  CACHE FILEPATH "" FORCE )
+if(Qt_version VERSION_LESS 5.0.0)
+  set(QT_QMAKE_EXECUTABLE ${fletch_BUILD_INSTALL_PREFIX}/bin/qmake
+    CACHE FILEPATH "" FORCE )
 
-file(APPEND ${fletch_CONFIG_INPUT} "
+  file(APPEND ${fletch_CONFIG_INPUT} [[
```

Comment:
Not sure if this is a typo - using `[[` instead of `"`

Is it supposed to work?

Reply:
> Not sure if this is a typo

Nope; intentional.

> Is it supposed to work?

[Yes](https://cmake.org/cmake/help/v3.0/manual/cmake-language.7.html#bracket-argument).

Reply:
Neat. Thanks for pointing it out.